frontend/src/metabase/search/analytics.ts

```diff
@@ -0,0 +1,53 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+import type { SearchRequest, SearchResponse } from "metabase-types/api";
+
+type SearchRequestFilter = Pick<
```

Comment:
I might be missing some background but why not "filtered search request" or something